tools/Google.Cloud.Tools.ReleaseManager/AddCommand.cs

```diff
@@ -123,5 +128,29 @@ protected override void ExecuteImpl(string[] args)
             Console.WriteLine($"Added {id} to the API catalog with the following configuration:");
             Console.WriteLine(api.Json.ToString(Formatting.Indented));
         }
+
+        private static Api.Service ParseServiceConfigYaml(string path)
+        {
+            if (path is null)
+            {
+                return null;
+            }
+
+            var deserializer = new Deserializer();
+            using (var reader = File.OpenText(path))
```

Comment:
Should we check if path is valid and if the file exists, before reading?

Reply:
If it's not, generation will fail when we try to use it in what's inevitably the next command anyway - better to fail at this point, I'd say :)